src/main/java/org/sonarlint/intellij/config/global/SonarLintGlobalSettings.java

```diff
@@ -62,6 +105,18 @@ public SonarLintGlobalSettings getState() {
   @Override
   public void loadState(SonarLintGlobalSettings state) {
     XmlSerializerUtil.copyBean(state, this);
+    if(includedRules != null) {
```

Comment:
Formatting issue. Please ensure you are using the SonarSource configuration, and apply a reformat.